spec/api-crash-reporter-spec.js

```diff
@@ -15,15 +15,15 @@ const { app, BrowserWindow, crashReporter } = remote
 const { expect } = chai
 chai.use(dirtyChai)
 
-describe('crashReporter module', () => {
-  if (process.mas || process.env.DISABLE_CRASH_REPORTER_TESTS) return
-
-  // TODO(alexeykuzmin): [Ch66] Fails. Fix it and enable back.
-  if (process.platform === 'linux') return
+const ifdescribe = (condition) => {
+  return condition ? describe : describe.skip
+}
 
+// TODO(alexeykuzmin): [Ch66] Fails on linux. Fix it and enable back.
+ifdescribe(!process.mas && process.platform !== 'linux')('crashReporter module', () => {
   let originalTempDirectory = null
   let tempDirectory = null
-  const specTimeout = 180000
+  const specTimeout = 10000
```

Comment:
The tests still seem to be flaky on CI, as you can see in the CI results of this PR.

Reply:
I don't think the results are flaky, I think they're legitimately failing. cc @nitsakh